src/stcal/jump/jump.py

```diff
@@ -1037,20 +1153,37 @@ def find_faint_extended(
             extended_emission[exty, extx] = 1
 
             #  find the contours of the extended emission
-            contours, hierarchy = cv.findContours(extended_emission, cv.RETR_EXTERNAL, cv.CHAIN_APPROX_SIMPLE)
+            contours = skimage.measure.find_contours(extended_emission)
             #  get the contours that are above the minimum size
-            bigcontours = [con for con in contours if cv.contourArea(con) > min_shower_area]
+            bigcontours = [con for con in contours if area_of_polygon(con) > min_shower_area]
             #  get the minimum enclosing rectangle which is the same as the
             # minimum enclosing ellipse
-            ellipses = [cv.minAreaRect(con) for con in bigcontours]
+            rectangles = [
+                minimum_bounding_rectangle(con) for con in bigcontours
+            ]
+            ellipses = [
+                (
+                    tuple(np.mean(rectangle[[0, 2], :], axis=0)),
+                    tuple(np.hypot(*np.diff(rectangle[[0, 1, 2], :], axis=0))),
+                    np.degrees(np.arctan2(*np.flip(np.diff(rectangle[[3, 0], :], axis=0)[0]))),
+                )
+                for rectangle in rectangles
+            ]
```

Comment:
this is where the axis differences comes from; the minor and major radii are larger than those from `opencv` by 1.5 (?) units